java/src/com/here/flexpolyline/PolylineEncoderDecoder.java

```diff
@@ -1,8 +1,18 @@
 /*
- * Copyright (C) 2019 HERE Europe B.V.
- * Licensed under MIT, see full license in LICENSE
- * SPDX-License-Identifier: MIT
- * License-Filename: LICENSE
+ * Copyright 2019 HERE Europe B.V
```

Comment:
In all the other implementations the license header looks like
```
/*
 * Copyright (C) 2019 HERE Europe B.V.
 * Licensed under MIT, see full license in LICENSE
 * SPDX-License-Identifier: MIT
 * License-Filename: LICENSE
 */
```
Please stick with that convention, it is useful for parsing the license requirements.

Reply:
Done. Thanks!